docs/deploy.md

```diff
@@ -0,0 +1,119 @@
+# Deploying M3DB
+
+## Deploying a single node
```

Comment:
Not sure if this entire file should just be called "Getting started" or not. This doesn't quite feel like a full "deploy"